src/lambda_function.py

```diff
@@ -82,21 +103,24 @@ def response(*actions):
 # For new incoming calls, play greeting and collect digits of destination number.
 # Regex for digits entered allows US calling, except for premium rate numbers
 def new_call_handler(call_id):
-        logger.info('SEND {} {}'.format(log_prefix, 'Sending PlayAndGetDigits action to get Destination Number'))
-        return response(pause_action(call_id), play_and_get_digits_action(call_id, '^(?!1900)1[0-9][0-9][0-9](\d{7})$', 'welcome_vf_demo.wav', 'invalid_entry.wav'))
+    logger.info('SEND {} {}'.format(
```

Comment:
Is this just a linter change?  Or were carriage returns added intentionally?

Reply:
linter